core/src/main/java/org/frankframework/jdbc/MessageStoreListener.java

```diff
@@ -181,6 +182,8 @@ public Message extractMessage(@Nonnull RawMessageWrapper<Serializable> rawMessag
 		// If not, then the RawMessageWrapper context still contains some info we want to retain, such as MID, CID and Storage Key.
 		// So copying it here to thread context is always the right thing.
 		context.putAll(rawMessageWrapper.getContext());
+		context.remove(JdbcListener.ADDITIONAL_QUERY_FIELDS_KEY);
```

Comment:
Wat is dit precies en waarom gooi je het weg?

Reply:
Ik verzamel alle extra velden uit de query in een map, en die map stop ik als map in de context van de rawMessage onder deze key `ADDITIONAL_QUERY_FIELDS_KEY`. Op die manier heb ik ze allemaal bij elkaar, en kan ik ze makkelijk aan de session toevoegen zonder de session te "bevuilen" met wat nog meer in de context zit. (JdbcTableListener zet verder niks uit de context in de session; dat doet alleen de MessageStoreListener).

Maar omdat de MessageStoreListener al een "putAll" doet van de hele context staat ook deze extra map erin. Dus moet die er weer uit. :-(

Reply:
Dit zou je eigenlijk dan moeten asserten in een test.